mock/databaseWriterStub.go

```diff
@@ -39,7 +39,7 @@ func (dwm *DatabaseWriterStub) DoMultiGet(query map[string]interface{}, index st
 	return nil, nil
 }
 
-// DoBulkRemove -
+// DoBulkRemove
```

Comment:
Please revert this

Reply:
yes please add the"-" after so that the linter does not complain

Reply:
done.